Doc/tools/extensions/refwarn.py

```diff
@@ -0,0 +1,58 @@
+"""Capture and report missing cross-references in the documentation."""
+
+import csv
+import os.path
+from collections import Counter
+
+
+def _rpr(obj, /):
```

Comment:
I still wasn't sure what this was doing, can we be more explicit?

```suggestion
def _repr(obj, /):
```